src/features/Header/Logo/Logo.tsx

```diff
@@ -0,0 +1,15 @@
+import { Button } from "@/components/Button/Button";
+import { ButtonAppearance } from "@/components/Button/Button.types";
+import Image from "next/image";
+import Styles from "./Logo.module.scss";
+import Link from "next/link";
+
+export const Logo = () => {
+  return (
+    <div className={Styles.logo}>
+      <Link className={Styles.link} href={"/"}>
+        <Image src={"/twitter.svg"} alt="Twitter Logo" width={30} height={30} />
+      </Link>
```

Comment:
i think its better to move width, height to scss. use this variant only if its dynamic values